cmd/vic-machine/validate/network.go

```diff
@@ -22,36 +22,37 @@ import (
 	"github.com/vmware/vic/pkg/errors"
 )
 
-func (v *Validator) createBridgeNetwork() error {
+func (v *Validator) createBridgeNetwork() (bool, error) {
```

Comment:
Sure you saw but I've moved the network create into the lib portion - that way it's only created once we've actually validated the config for the appliance.